heatequation.jl

```diff
@@ -0,0 +1,71 @@
+using OrdinaryDiffEq, ModelingToolkit, DiffEqOperators, DomainSets
```

Comment:
Since you've added the file with the different name, you can also remove this file.